package.json

```diff
@@ -39,8 +39,10 @@
     "test:a11y": "yarn playwright test --config tests/a11y/playwright.config.ts",
     "test:unit": "yarn jest --config=tests/unit/jest.config.js",
     "test:unit:update": "yarn test:unit --updateSnapshot",
-    "test:browser": "docker run --rm --network host -v $(pwd):/work/ -w /work/ mcr.microsoft.com/playwright:v1.25.0-focal yarn playwright test --config tests/browser/playwright.config.ts",
-    "test:browser:update": "yarn build:storybook && docker run --rm --network host -v $(pwd):/work/ -w /work/ mcr.microsoft.com/playwright:v1.25.0-focal yarn playwright test --config tests/browser/playwright.config.ts -u",
+    "test:browser": "yarn test:browser:interaction && yarn test:browser:all",
+    "test:browser:all": "docker run --rm --network host -v $(pwd):/work/ -w /work/ mcr.microsoft.com/playwright:v1.30.0-focal yarn playwright test --config tests/browser/playwright.config.ts",
```

Comment:
Are these command names flipped? Should `:all` be the one to run both the normal browser tests and the interaction tests?

Reply:
That does make sense, I didn't want to mess with any CI changes.